package.json

```diff
@@ -35,14 +35,18 @@
     "node-schedule": "^1.3.2",
     "pino": "^6.7.0",
     "pino-pretty": "^4.3.0",
-    "soundex-code": "^1.0.5"
+    "remark": "^13.0.0",
+    "soundex-code": "^1.0.5",
+    "stopword": "^1.0.6",
+    "strip-markdown": "^4.0.0"
```

Comment:
[stopword](https://www.npmjs.com/package/stopword) does what @stemount explained in this comment https://github.com/EddieJaoudeCommunity/EddieBot/issues/183#issuecomment-674404453
>Helper function to strip a question of words that are not that useful.

It should help in getting a better search result, but I'm not sure if it does 😁.
what do you think, should I remove it or not?

---
I'm using [strip-markdown](https://www.npmjs.com/package/strip-markdown) to remove any markdown syntax from the issue body before wrapping it in a markdown link to avoid any issue like this: ( `[Git[Hub](https://github.com)` => [Git[Hub](https://github.com) )
but strip-markdown is a plugin for a library called [remark](https://www.npmjs.com/package/remark) so I installed it to use strip-markdown.

But I found another literary [remove-markdown](https://www.npmjs.com/package/remove-markdown) that does the same without  requiring any dependencies, I will use this one instead.

Reply:
Thank you so much for the explanation 👍. I agree with you in using all these dependencies, and the remove-markdown package seems fine too 😃